job-service-db/src/main/resources/db/migration/V2__job_service.sql

```diff
@@ -0,0 +1,54 @@
+--
```

Comment:
I presume there's a good reason for starting with V2?  I think we discussed this before and there was.  Remind me what it is.